File: java/src/LC_27_RemoveElement.java
// https://leetcode.com/problems/remove-element/description
public class LC_27_RemoveElement {

    public int removeElement(int[] nums, int val) {
        int index = 0;
        for (int num : nums) {
            if (num != val)
                nums[index++] = num;
        }
        return index;
    }
}

File: python/LC_27_RemoveElement.py
# https://leetcode.com/problems/remove-element/description
from typing import List

class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        index = 0
        for num in nums:
            if num != val:
                nums[index] = num
                index +=1
        return index
        